### PR TITLE
OCPBUGS-14810: Chore: Update OWNERS and OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by mpatlasov**
Changing the component as per the other drivers and adding Maxim in storage maintainers.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -5,3 +5,4 @@ aliases:
     - gnufied
     - bertinatto
     - dobsonj
+    - mpatlasov
```
Comment (RomanBednar) [inline OWNERS_ALIASES:8]: @jsafrane @mpatlasov What about adding all team members while doing this chore? Same for other PRs that are currently opened.

Comment (mpatlasov) [reply OWNERS_ALIASES:8]: @RomanBednar , I really don't know which path is "correct": adding all team members vs. each team member adds himself/herself as needed. Let's wait for Jan's comment and we also may discuss it with the team online.

Comment (mpatlasov) [reply OWNERS_ALIASES:8]: @RomanBednar , we discussed this topic with Jan today. I was wrong assuming that each engineer has to add himself/herself to proper OWNERS files. You're right -- all team members must be in each OWNERS file. I'll update my PRs soon accordingly.

Comment (mpatlasov) [reply OWNERS_ALIASES:8]: @RomanBednar , we discussed this topic with Jan today. I was wrong assuming that each engineer has to add himself/herself to proper OWNERS files. You're right -- all team members must be in each OWNERS file. I'll update my PRs soon accordingly.

Comment (RomanBednar) [reply OWNERS_ALIASES:9]: Awesome, thank you @mpatlasov!

**Pull request merged**